yt/visualization/fixed_resolution.py

```diff
@@ -614,6 +616,39 @@ def __getitem__(self, item):
             north_vector=dd.north_vector,
             method=dd.method,
         )
+        if self.data_source.moment == 2:
+
+            def _sq_field(field, data, fname: Tuple[str, str]):
+                return data[fname] ** 2
```

Comment:
I'm confused; I thought in most places, `fname` is meant to designate a single `str`, not the complete field tuple

Reply:
Yes, but the problem here is that there's a name collision--`field` is already in use in the function signature. If you've got any other suggestions I'm all ears

Reply:
I think `item` would make more sense in this context; Also, just curious, I don't think I've ever seen an example of a derived field where the first argument `field` was used, do you happen to know when it's useful ?